Test/CoreSDK.Test/TestFramework/Shared/StubTelemetryProcessor2.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.ApplicationInsights.TestFramework
```

Comment:
or if that is not possible, create a private class for the specific test case... in the test framework we only need 1 stub...